config/routes.rb

```diff
@@ -1,6 +1,11 @@
 # rubocop:disable Metrics/BlockLength
 
 Rails.application.routes.draw do
+  constraints(host: /www2\.|\.education\./) do
```

Comment:
```suggestion
  constraints(host: /^www2\.|\.education\./) do
```

In case we were to implement this in the app, should probably lock `www2` for the sake of clarity to the reader if nothing else.